wgsl/index.bs

```diff
@@ -40,11 +40,14 @@ thead {
 
 ## Goals ## {#goals}
 
- * Trivially convertable to SPIR-V
- * Constructs are defined as normative references to their SPIR-V counterparts
- * All features in [SHORTNAME] are directly translatable to SPIR-V. (No polymorphism, no general pointers, no overloads, etc)
- * Features and semantics are exactly the ones of SPIR-V
- * Each item in this spec *must* provide the mapping to SPIR-V for the construct
+* Faithfully and simply converted to and from SPIR-V
+* Faithfully and simply converted to Metal Shading Language
+* Faithfully and simply converted to HLSL
+* Suitable for a human to read and write
+* Constructs are defined as normative references to their SPIR-V counterparts
+* All features directly translatable to SPIR-V. (No polymorphism, no general pointers, no overloads, etc)
+* Semantics match SPIR-V opcodes, where applicable
+* Each construct in this spec *must* provide the mapping to SPIR-V
```

Comment:
Does it also need to provide the mapping _from_ SPIR-V? Or is "mapping to" meant to indicate mappings in both directions?